File: TemplateMapper/TemplateMapper/Models/DrawingLayer.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public partial class DrawingLayer
    {
        public long DrawingLayerId { get; set; }
        public string Name { get; set; }
        public long DrawingId { get; set; }

        public virtual Drawing Drawing { get; set; }
    }
}

File: DataSummitSolution/DataSummitUI/src/app/efModels/drawingLayer.ts
import { Drawing } from "../drawings/models/drawing";

export class DrawingLayer {

	DrawingLayerId: number;
	Name: string;
	DrawingId: number;
	Drawing: Drawing;

	constructor(drawingLayerId?: number, name?: string, drawingId?: number,
		drawing?: Drawing)
	{
		this.DrawingLayerId = drawingLayerId || 0;
		this.Name = name || "";
		this.DrawingId = drawingId || 0;
		this.Drawing = drawing;
	}
}
